ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -140,10 +147,26 @@
       },
     },
     {
-      id: 'E6S Conflag Strike Knockback',
-      regex: Regexes.startsUsing({ source: 'Raktapaksa', id: '4C10', capture: false }),
+      id: 'E6S Downburst Knockback 1',
+      regex: Regexes.startsUsing({ source: 'Raktapaksa', id: '4BFB', capture: false }),
       response: Responses.knockback(),
     },
+    {
+      id: 'E6S Downburst Knockback 2',
+      regex: Regexes.startsUsing({ source: 'Raktapaksa', id: '4BFC', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'E6S Conflag Strike',
+      regex: Regexes.startsUsing({ source: 'Raktapaksa', id: '4C10', capture: false }),
+      infoText: {
+        en: 'Conflag',
```

Comment:
I'm not sure that "Conflag" on its own is super useful? Maybe say "go to spots for chains?" or something?